src/legacy/store/GraphQLStoreQueryResolver.js

```diff
@@ -246,8 +243,8 @@ class GraphQLStoreSingleQueryResolver {
       if (subscribedIDs) {
         // always subscribe to the root ID
         subscribedIDs[nextID] = true;
-        this._subscription = GraphQLStoreChangeEmitter.addListenerForIDs(
-          Object.keys(subscribedIDs),
+        this._subscription = this._storeData.getChangeEmitter().addListenerForIDs(
+          Object.keys(subscribedIDs).map(id => this._getCanonicalID(id)),
```

Comment:
Right, so `GraphQLChangeEmitter` used to map range -> canonical ID for in `addListenerForIDs`, so moving the logic here just keeps the old behavior intact.